tools/distrib/python/submit.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import shutil
+import subprocess
+import tempfile
+
+parser = argparse.ArgumentParser(
+    description='Submit the package to a PyPI repository.')
+parser.add_argument(
+    '--repository', metavar='r', type=str, default='pypi',
+    help='The repository to push the package to. '
+         'Ensure the value appears in your .pypirc file. '
+         'Defaults to "pypi".'
+)
+args = parser.parse_args()
+
+# Move to the directory of this file.
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Even better, don't do this at all, but pass the right directory (the one containing cwd) as the 'cwd' argument to subprocess.call.